libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts

```diff
@@ -65,6 +68,8 @@ export const DocumentProviderOnboarding: Form = buildForm({
               id: 'applicant.phoneNumber',
               name: m.applicantPhoneNumber,
               variant: 'tel',
+              format: '###-####',
+              placeholder: '000-0000',
```

Comment:
This makes it so we can only use icelandic phonenumbers. Might be some case where there is a phonenumber from different country ? Open for disscussion

Reply:
True, good point. We will have to think about that later on and maybe change it